app/controllers/uploads_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UploadsController < ApplicationController
```

Comment:
Should we have separate controllers for uploads and downloads or combine them into one?